docs/guides/Integration-Guides/payments-integration-guide/google-pay-processing-information-for-payment-providers-and-anti-fraud.md

```diff
@@ -5,9 +5,40 @@ hidden: false
 createdAt: "2022-06-22T00:00:00.000Z"
 updatedAt: ""
 ---
+Here are some answers to questions you may have about transactions with Google Pay:
 
-Purchase transactions made through Google Pay are sent to payment providers and anti-fraud with the same payload structure present in a purchase made through credit or debit cards on VTEX.
+## 1. What are the differences between the payload used for a standard credit card payment and the one used for Google Pay [DPAN](https://help.vtex.com/en/tutorial/dpan-and-fpan-understanding-security-in-the-online-tokenized-payment-flow--3RM7RvhKZ057wja5xVEOqb)?
 
-The payment method indicated in the transaction is the card brand used in the purchase. If the provider wishes to have the "Google Pay" information present in the transaction, it is necessary to request it from [VTEX Support](https://help.vtex.com/support). After implementing the request, the `paymentOrigin` field will be sent filled in as "Google Pay" whenever the transaction is made through this digital wallet.
+Besides the original fields for a credit or debit card payment transaction, the Google Pay DPAN transaction has three additional fields:
 
-Similarly, when providers choose to process DPAN cards, after contacting VTEX Support, transactions will be sent with two additional fields: `cryptogram` and `eci` (filled only in purchases with Visa cards).
+- `cryptogram`: 3DS cryptogram information (previously called `3DS_criptogram`).
+- `eci`: Electronic Commerce Indicator (previously called `ECI_Indicador`). This field is sent only for Visa transactions and indicates the result of the authentication attempt made by the 3DS system.
+- `paymentOrigin` (optional): Identifies the type of card used for payment. To enable this field in Google Pay (DPAN) transactions or FPAN transactions and receive the information in the [Create payment endpoint](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments) payload, open a ticket with [VTEX Support](https://help.vtex.com/support).
+
+In the VTEX Admin, go to **Orders > Transactions** to check if a transaction has been made with Google Pay.
+
+![DPAN transaction fields](https://raw.githubusercontent.com/vtexdocs/dev-portal-content/main/docs/guides/Integration-Guides/payments-integration-guide/dpan-transaction-fields-1.png)
+
+> ℹ️ The `paymentOrigin`, `panType`, and `cryptogram` fields aren't sent in standard credit or debit card transactions (without using a digital wallet).
+
+## 2. What are the possible values for the "paymentOrigin" field in the payload?
+
+The "paymentOrigin" field is always sent with the "Google Pay" value. Payment transactions using other payment methods don't have the "paymentOrigin" field in the payload.
+
+## 3. Is the card CVV sent in Google Pay transactions?
```

Comment:
Use the abreviation only after you have used the complete spelled out version before. E.g., "_Federal Bureau of Investigation (FBI)_"

Reply:
Great! Done!